src/cljc/rems/common/form.cljc

```diff
@@ -224,6 +226,11 @@
       remove-empty-keys
       nil-if-empty))
 
+(defn validate-fields-compatibility [form languages]
+  (-> {:form/fields (validate-fields (:form/fields form) languages)}
+      remove-empty-keys
+      nil-if-empty))
```

Comment:
is this function unused? if you want to keep it, a docstring would be nice

Reply:
Should be fixed by now